src/driver/test/service-architecture/SecondaryAllowed.yml

```diff
@@ -0,0 +1,57 @@
+SchemaVersion: 2018-07-01
+Pool:
+    QueryOptions:
+        maxPoolSize: 1000
+        socketTimeoutMS: 5000
+        connectTimeoutMS: 5000
+Actors:
+    - Name: StepdownCommand
+      Type: AdminCommand
+      Threads: 1
+      ExecutionStrategy:
+          Retries: 0
+          ThrowOnFailure: false
+      Phases:
+        - NoOp: true
+        - Phase: 1
+          Repeat: 1
+          Operation:
+              OperationMetricsName: "Stepdown"
+              OperationName: AdminCommand
+              OperationSleepBefore: 30 seconds
+              OperationCommand:
```

Comment:
It's probably also worth attaching an `OperationSleepAfter` because the option awaits the stepdown but doesn't necessarily go till the stepup.

Reply:
> but doesn't necessarily go till the stepup.

should it? is that a shortcoming of the RunCommand actor?

Reply:
So kinda. I think that waiting for a proper StepUp is a much more programatically complex task that requires looping until we can get a proper primary again. The StepDown itself means a handoff of primary status that will thrash readPreference like we expect. What I'm trying to say is that I think this is good enough for now even if there is some fudge in the box.